README.md

```diff
@@ -1,16 +1,226 @@
-# Title
+# Programming as Conversation 3: Methods on Ruby's Built-In Objects
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Define Object Oriented Programming (OOP)
+- Define instance
+- Define class
+- Define attribute / property
+- Define "dot-notation" syntax
+- Define method
+- Demonstrate method chaining
+- Find documentation about methods and attributes
 
 ## Introduction
 
-## SWBAT 1
+In the previous lessons, you've learned to encapsulate work in methods. Methods
+can take in arguments and then use repetition, selection, and expression
+evaluation to provide information about the arguments they were handed.
 
-## SWBAT 2
+But Ruby is an Object-Oriented Language. That means that many of the default
+types of data, can run methods that are "inside" them. They can tell you things
+about themselves or they can transform themselves. We've already seen that the
+constants of `3.14` or `8675309` or `"Love Will Tear Us Apart"` can tell you
+things about themselves.
+
+```ruby
+3.14.class #=> Float
+```
+
+This. Is. So. Cool.
+
+In time, you'll learn to create objects of your own, things that represent
+`Student`s or `PayrollRecord`s. This is known as Object-Oriented Programming.
+This is a vast topic that will take many lessons and a lot of practice to
+learn. Our point here is to show you how to interact with the default objects
+Ruby provides. But don't worry, you'll _definitely_ learn to teach the objects
+of your world to tell you about themselves soon enough.
+
+## Define Object-Oriented Programming
+
+Bundling code into little objects like this that are "smart" is "Object-Oriented
+Programming (OOP)." OOP is a huge topic that will take a lot of practice to
+master. However, for the moment we merely want to show you how to use the
+objects that Ruby provides to help you do work.
+
+## Define Instance
+
+Sometimes programmers call "objects" instances because they are instances of a
+class. `3.14` is an instance of the `Float` class. `"hi"` is an instance of
+the `String` class.
+
+## Define Class
+
+A class is the thing that makes all instances the same thing. Remember our
```

Comment:
This first setence isn't quite clear. I think the meaning can be inferred in context but read alone I'm not sure what it is saying